<launch>
    <node pkg="idmind_sensorsboard" type="idmind_sensors.py" name="idmind_sensors" output="screen">
    </node>
</launch>